src/CssInliner.php

```diff
@@ -315,6 +324,20 @@ public function setDebug(bool $debug): self
         return $this;
     }
 
+    /**
+     * Configure whether to remove the !important annotation from inline styles or not.
+     * Must be used before inlineCss is called.
+     *
+     * @param bool $removeImportantAnnotationFromAllInlineStyles set to true to remove the !important annotation
+     * @return self
```

Comment:
I think the `phpdoc_separation` seperation error is coming from here, and needs a empty PHPDoc line between `@param` and `@return`.